desktop-exporter/app/routes/trace-view.tsx

```diff
@@ -64,10 +81,10 @@ End: ${span.endTime}
 }
 
 export default function TraceView() {
-  const traceData = useLoaderData();
-  const [selectedSpanID, setSelectedSpanID] = React.useState(
-    traceData.spans[0].spanID,
-  );
+  const traceData = useLoaderData() as TraceData;
```

Comment:
I was not able to get types to flow from the loader function, through the router, through `useLoaderData()`, but we can assert the type here and get typing through the rest of the app.